quinn-proto/src/connection.rs

```diff
@@ -2452,12 +2466,17 @@ impl Connection {
         self.streams.alloc_remote_stream(self.side, ty);
     }
 
+    /// Accept a remotely initiated stream if possible
+    ///
+    /// Returns `None` if this connection does not currently allow the peer to open streams in
+    /// this direction.
```

Comment:
This is inaccurate; `accept` returns `None` iff there are no new incoming streams.